common/services/person.js

```diff
@@ -11,13 +11,32 @@ function getFullname ({
 
 function format (data) {
   const relationships = ['gender', 'ethnicity']
-
-  return mapValues(data, (value, key) => {
-    if (relationships.includes(key) && typeof value === 'string') {
-      return { id: value }
+  const identifiers = [
+    'pnc_number',
+    'cro_number',
+    'prison_number',
+    'niche_reference',
+    'athena_reference',
+  ]
```

Comment:
We changed the first two of these keys to avoid the acronyms, see https://github.com/ministryofjustice/hmpps-book-secure-move-api/pull/77/commits/65c1eed29126f3b01a885eca1d1668b9754d1b5d

Reply:
Ah okay. I’ll update them. 

Bit of inconsistency with those names now though :/ 

_number_ was dropped from `pnc_number` and `cro_number` but then _number_ is used in `prison_number` and _reference_ is used in `athena_reference` and `niche_reference`.

Reply:
Maybe it just be?

```
'police_national_computer'
'criminal_records_office'
'nomis’
'niche'
'athena'
```

Reply:
yes that might be better, if everybody agrees I'm happy to change the others too